openfisca_uk/variables/government/dwp/pension_credit/pension_credit.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_uk.model_api import *
+
+
+class pension_credit(Variable):
+    label = "Pension Credit"
+    entity = BenUnit
+    definition_period = YEAR
+    value_type = float
+    unit = "currency-GBP"
+    reference = "https://www.legislation.gov.uk/ukpga/2002/16/contents"
+
+    def formula(benunit, period, parameters):
+        gc = benunit("guarantee_credit", period)
+        sc = benunit("savings_credit", period)
+        eligible = benunit("pension_credit_eligible", period)
+        would_claim = benunit("would_claim_pc", period)
+        return (eligible & would_claim) * (gc + sc)
```

Comment:
```suggestion
        return eligible * would_claim * (gc + sc)
```

Reply:
I thought we opted for & over * where it's a boolean operation?